packages/client/package.json

```diff
@@ -32,11 +32,12 @@
     "@spectrum-css/tag": "^3.1.4",
     "@spectrum-css/typography": "^3.0.2",
     "@spectrum-css/vars": "^3.0.1",
-    "apexcharts": "^3.22.1",
+    "apexcharts": "^3.48.0",
     "dayjs": "^1.10.8",
     "downloadjs": "1.4.7",
     "html5-qrcode": "^2.2.1",
     "leaflet": "^1.7.1",
+    "lodash": "^4.17.21",
```

Comment:
I don't think it's necessary to add in lodash to the client.

Reply:
It's used in the client but not explicitly specified in the package.json, and it's not specified at the root level either, made sense to me to add it 🤷

Reply:
Will remove